README.md

```diff
@@ -57,5 +82,239 @@ Due to the 32 characters limitation, it can be truncated.
+      application: app-ingress-controller
+  template:
+    metadata:
+      labels:
+        application: app-ingress-controller
+    spec:
+      containers:
+      - name: controller
+        image: registry.opensource.zalan.do/teapot/kube-aws-ingress-controller:latest
+        env:
+        - name: AWS_REGION
+          value: $YOUR_REGION
+```
+
+The image used can be customized, we provide `registry.opensource.zalan.do/teapot/kube-aws-ingress-controller:latest` as a public usable docker image built from this codebase.
+
+Additionally to the ingress controller, we use [skipper](https://github.com/zalando/skipper) to route the traffic to the application, in a setup that follows what is described [here](https://kubernetes-on-aws.readthedocs.io/en/latest/user-guide/ingress.html).
+
+We deploy `skipper` as a `DaemonSet` using the following yaml: 
+
+```
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: skipper-ingress
+  namespace: kube-system
+  labels:
+    application: skipper-ingress
+    version: v0.9.19
+    component: ingress
+  annotations:
+    daemonset.kubernetes.io/strategyType: RollingUpdate
+    daemonset.kubernetes.io/maxUnavailable: "1"
+spec:
+  selector:
+    matchLabels:
+      application: skipper-ingress
+  template:
+    metadata:
+      name: skipper-ingress
+      labels:
+        application: skipper-ingress
+        version: v0.9.19
+        component: ingress
+    spec:
+      hostNetwork: true
+      containers:
+      - name: skipper-ingress
+        image: registry.opensource.zalan.do/teapot/skipper:v0.9.19
+        ports:
+        - name: ingress-port
+          containerPort: 9999
+          hostPort: 9999
+        args:
+          - "skipper"
+          - "-application-log-level=DEBUG"
+          - "-kubernetes"
+          - "-kubernetes-in-cluster"
+          - "-address=:9999"
+          - "-proxy-preserve-host"
+          - "-serve-host-metrics"
+        resources:
+          limits:
+            cpu: 200m
+            memory: 200Mi
+          requests:
+            cpu: 25m
+            memory: 25Mi
+```
+
+To complete the setup, the following is also required: 
+
+- create an additional security group to allow traffic from the internet to the load balancers. This can be done by using the following cloud formation stack which needs to have the same name of the autoscaling group used for the nodes:
+```
+Resources:
+  IngressLoadBalancerSecurityGroup:
+    Properties:
+      GroupDescription: {Ref: 'AWS::StackName'}
+      SecurityGroupIngress:
+      - {CidrIp: 0.0.0.0/0, FromPort: 80, IpProtocol: tcp, ToPort: 80}
+      - {CidrIp: 0.0.0.0/0, FromPort: 443, IpProtocol: tcp, ToPort: 443}
+      VpcId: "$VPC_ID"
+      Tags:
+        - Key: "ClusterID"
+          Value: "$CLUSTER_ID"
+    Type: AWS::EC2::SecurityGroup
+```
+
+- Make the port used by `skipper` (in this example, port `9999`) accessible from the IP range of the VPC to allow health checking from the Application Load Balancer. 
```

Comment:
I would prefer to keep this minimal. This is the README for the ingress controller and I would leave to other documents (skipper doc itself?) this description.